### PR TITLE
I mean !set stream

### PR DESCRIPTION
**Pull request by irdumbs**
new PR from #300
game_name instead of status as per discussion

### CODE REVIEW

File: cogs/owner.py [hunk 1]
```diff
@@ -284,6 +284,27 @@ def __unload(self):
...
         await self.bot.say("Done.")
+        
+    @_set.command(pass_context=True)
+    @checks.is_owner()
+    async def stream(self, ctx, stream_name=None, *, game_name=None):
+        """Sets Red's streaming status
+
+        Leaving both stream_name and game_name empty will clear it."""
+
+        if status:
```
Comment (tekulvw) [inline cogs/owner.py]: Uh...you might've missed a few references

**Comment by irdumbs** [comment]
annd we're to 
[streamer] [stream_title]
stream_title because discord picks up the stream title, not the game.

**Comment by jaw3l** [comment]
what about YouTube streams? is this possible?

**Comment by irdumbs** [comment]
discord doesn't support that atm

**Pull request merged**